packages/appstore/src/components/wallet-header/wallet-header.tsx

```diff
@@ -16,10 +16,9 @@ type TWalletHeader = {
 
 const WalletHeader = observer(({ wallet_account }: TWalletHeader) => {
     const { client, traders_hub } = useStore();
```

Comment:
Is traders_hub used anywhere in this file

Reply:
will remove it